### PR TITLE
Discord link fix: make permanent (never expire)

### PR DESCRIPTION
**Pull request by limonspb**
Forgot that by default discord invite link is being expired after 7 days.
Changing the link to a permanent one.

### CODE REVIEW

**Review by blckmn — CHANGES_REQUESTED** [review]
You can use

HTTPS://discord.betaflight.com/invite

**Review by KarateBrot — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
